test/main.cpp

```diff
@@ -30,7 +34,51 @@
 
 using namespace std;
 
+static void set_filters_for_backend() {
+  std::string backend = "GENERIC";
+  char* pBE = getenv("NGRAPH_TF_BACKEND");
+  if (pBE) {
+    backend = std::string(pBE);
+    // backend.replace(backend.begin(), backend.end(), ":", "");
+    backend = std::regex_replace(backend, std::regex(":"), "");
+  }
+  std::string this_filepath = __FILE__;
+  this_filepath =
+      std::regex_replace(this_filepath, std::regex("^(.*)/.*$"), "$1");
+  std::string filter_file =
+      this_filepath + "/" + "cpp_tests_list_" + backend + ".txt";
+  fstream fs;
+  fs.open(filter_file, ios::in);
+  if (fs.is_open()) {
+    std::string filters = "";
+    std::string line;
+    bool excluded_section = false;
+    while (getline(fs, line)) {
+      line = std::regex_replace(line, std::regex("^\\s+"), "");
+      line = std::regex_replace(line, std::regex("#.*$"), "");
+      line = std::regex_replace(line, std::regex("\\s+$"), "");
+      if (line.empty()) continue;
+      if (!excluded_section && line.find("[EXCLUDED]") == 0) {
+        if (filters.back() == ':') filters.pop_back();  // remove last :
+        filters += "-";
+        excluded_section = true;
+        continue;
+      }
+      if (excluded_section) {
+        if (line.at(0) == '-') line = line.erase(0, 1);
+      }
+      filters += line + ":";
+    }
+    fs.close();
+    if (filters.back() == ':') filters.pop_back();  // remove last :
+
+    ::testing::GTEST_FLAG(filter) = filters;
+  }
+}
+
 int main(int argc, char** argv) {
+  set_filters_for_backend();
```

Comment:
I think it's probably better to be explicit about the manifest file here, or perhaps find a way to distinguish between which "bridge backend" is active.